third-party/realdds/include/realdds/dds-option.h

```diff
@@ -0,0 +1,56 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+
+#include <third-party/json_fwd.hpp>
+
+#include <string>
+#include <vector>
+
+
+namespace realdds {
+
+struct dds_option_range
+{
+    float min = 0.0f;
+    float max = 0.0f;
+    float step = 0.0f;
+    float default_value = 0.0f;
+};
+
+class dds_option
+{
+    std::string _name;
+    float _value = 0.0f;
+    dds_option_range _range;
+    std::string _description;
+
+    std::string _owner_name; //To identify device or stream having this option
```

Comment:
Like I said: I don't think this is the right solution. We'll have to change it in coming PRs.

Reply:
See other comments, [LRS-576](https://rsjira.intel.com/browse/LRS-576) will address this